package.json

```diff
@@ -202,7 +209,7 @@
     "express": "^4.16.3",
     "file-loader": "6.2.0",
     "fstream": "1.0.12",
-    "html-webpack-plugin": "4.5.1",
+    "html-webpack-plugin": "^5.0.0-beta.5",
```

Comment:
Storybook wouldn't start without using this beta version of `html-webpack-plugin`